solutions/devsprint-eduardo-bocato-1/SpaceApp/Modules/Features/Details/Scenes/LaunchDetails/Views/LaunchDetailsBodyView.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  LaunchDetailsBodyView.swift
+//  SpaceApp
+//
+//  Created by Razee Hussein-Jamal on 08/03/2022.
+//
+
+import SwiftUI
+
+struct LaunchDetailsBodyView: View {
+    let title: String
+    let description: String
+    let buttonTitle: String
+    
+    var body: some View {
+        VStack(alignment: .leading, spacing: 10) {
```

Comment:
**[SUGGESTION]**
Define a `Metrics` enum to keep the values like this `spacing`.

Example:
```swift
private enum Metrics {
    static itemsSpacing: CGFloat = 10
    static bodyPadding: CGFloat = 15
}
```

Then:
```suggestion
        VStack(alignment: .leading, spacing: Metrics. itemsSpacing) {
```